tests/examples/minimal/pom.xml

```diff
@@ -75,6 +75,14 @@
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>com.google.errorprone</groupId>
+            <artifactId>error_prone_annotations</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>io.grpc</groupId>
```

Comment:
Ideas why this is needed?

Reply:
Thanks for the review. There's a git commit message detailing this, but tl;dr it fixes the minimal example test.

Reply:
In that case this might belong in a separate PR? Is it failing on master or why did it start failing with the changes? The minimal example test is doing double-duty as a canary for misaligned dependencies, and I'm worried this change papers over that. Misaligned dependencies sometimes result in funny breakage for users depending on their maven import order of pulumi dips.

 I can fill you in on the back-story here.

Reply:
I'll paper-over the commit, but leave it in the history before squashing so the details are still in this PR.